src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties

```diff
@@ -385,6 +385,9 @@ manage_question_response.step=Step
 manage_question_response.status=Published
 manage_question_response.order=Order
 
+message.slot.determinant=From
```

Comment:
All property keys must be in english, please check all the keys

Reply:
sorry, I used a translator though as I wasn't sure how to qualify the nature of the word. and it's done in the hope that this translation will go down well.